packages/shared/src/contracts/ethers-typechain/Coordinator.ts

```diff
@@ -0,0 +1,2145 @@
+/* Autogenerated file. Do not edit manually. */
```

Comment:
When and how is this file autogenerated?

Reply:
This file is generated by `typechain` on `postinstall` and `build` steps.